lib/room.js

```diff
@@ -110,6 +110,10 @@ class Room extends EventEmitter {
         get() {
           return signaling.state;
         }
+      },
+      mediaRegion: {
```

Comment:
Please document this class property.

Reply:
Added to CHANGELOG.md !

Reply:
I believe this is where the public docs for Room.mediaRegion should live. If so, the docs should match: https://code.hq.twilio.com/pages/client/video-sdk-api/idl/room.html

Reply:
@PikaJoyce Please update the documentation before resolving the conversation.